lib/utilities.py

```diff
@@ -174,6 +175,13 @@ def pretty_date(time=False):
     return "%s %s ago" % (str(diff), diff_string)
 
 
+# See: https://github.com/MLTSHP/mltshp/pull/769
+def rfc822_date(time):
+    # Make sure the datetime object is timezone aware
+    if time.tzinfo == None or time.tzinfo.utcoffset(time) == None:
+        time.replace(tzinfo=timezone.utc)
+    return time.strftime("%a, %d %b %Y %H:%M:%S %Z")
```

Comment:
Let's remove the comment for the PR reference.

Also, the "Z" I was suggesting earlier was to use a literal `Z` instead of `%Z`.

I don't think the `parse` import is needed, is it? I don't see it being used.

The current build failure is coming from an invalid import of `timezone`, which doesn't exist for Python 2.

Reply:
You should be able to remove the `parse` and `timezone` imports. I'd simplify to:

```python
datetime_format = "%a, %d %b %Y %H:%M:%S"
# If time argument has no timezone information, assume UTC ("Z")
if time.tzinfo == None or time.tzinfo.utcoffset(time) == None:
    format = datetime_format + " Z"
else:
    format = datetime_format + " %Z"
return time.strftime(format)
```